alvr/client_openxr/src/interaction.rs

```diff
@@ -489,25 +489,37 @@ pub fn get_hand_motion(
             linear_velocity: to_vec3(velocity.linear_velocity),
             angular_velocity: to_vec3(velocity.angular_velocity),
         };
-
-        return (Some(hand_motion), None);
+        if velocity.linear_velocity.x != 0.0
+            && velocity.linear_velocity.y != 0.0
+            && velocity.linear_velocity.z != 0.0
```

Comment:
Why is this check needed? We don't care about velocities anyway.

Reply:
I'm not sure if it's pico specific, but i can't seem to get out of this if clause unless this is met (controllers do seem visually completely stopped)